testing/fuchsia/run_tests.py

```diff
@@ -128,9 +109,45 @@ def bundled_test_runner_of(target_id: str) -> BundledTestRunner:
       resolved_packages.add(new_package)
     else:
       resolved_packages.add(os.path.join(OUT_DIR, package))
+  return resolved_packages
+
+
+def build_test_cases(tests: dict) -> List[TestCase]:
```

Comment:
Can we add tests for this logic?

I think we'll need to manually add an entry to linux_fuchsia.json to run the test.

Reply:
Done. Trying.

Reply:
OK, I think it works now :)